### PR TITLE
Allow for numbers to be used in the arithmetic transform expression

### PR DESCRIPTION
**Pull request by danholdaway**
## Description

Allows numerals to be used in the arithmetic transform.

### CODE REVIEW

File: src/eva/transforms/transform_utils.py [hunk 1]
```diff
@@ -62,7 +62,7 @@ def replace_cgv(logger, collection, group, variable, *argv):
         else:
             arg_replaced.append(arg)
 
-    # Assert that the lenghts make sense
+    # Assert that the lengts make sense
```
Comment (CoryMartin-NOAA) [inline src/eva/transforms/transform_utils.py]: still spelled wrong :-)

File: setup.py [hunk 2]
```diff
@@ -39,6 +39,7 @@
         'cartopy>=0.18.0',
         'scikit-learn>=1.0.2',
         'xarray>=0.11.3',
+#        'seaborn==0.12',
```
Comment (CoryMartin-NOAA) [inline setup.py]: I think PEP8 is complaining about this, perhaps the comment needs to be in front of the quote?

**Review by CoryMartin-NOAA — COMMENTED** [review]
Two comments

File: src/eva/transforms/transform_utils.py (hunk 1 above)
Comment (danholdaway) [reply src/eva/transforms/transform_utils.py]: lol!

**Review by CoryMartin-NOAA — APPROVED** [review]
approved pending tests all pass

**Pull request merged**